packages/appstore/src/components/app-content.tsx

```diff
@@ -4,6 +4,7 @@ import { observer, useStore } from '@deriv/stores';
 import Routes from 'Components/routes/routes';
 import classNames from 'classnames';
 import './app.scss';
+import './override-other-packages-styles-to-make-the-code-even-more-unclear-because-everything-is-asap.scss';
```

Comment:
wait what 😂

Reply:
I guess this name is 100% suitable for this file :)

Reply:
loved the name ❤️ 😄